source/renderer/app/containers/wallet/WalletSettingsPage.js

```diff
@@ -79,7 +79,7 @@ export default class WalletSettingsPage extends Component<Props> {
         isLegacy={isLegacyWallet}
         walletId={activeWallet.id}
         walletName={activeWallet.name}
-        creationDate={creationDate}
+        creationDate={creationDate || new Date()}
```

Comment:
@tomothespian It was generating flow test error, that's why it was added.

Reply:
@yakovkaravelov that is weird. Can you investigate that? Maybe on some place `creationDate` type is defined with `?` and that is not correct because we will get a date from store.